Divd 2023 00040 (#685)
DIVD-NL/web-csirt: _cases/2023/DIVD-2023-00040.md

---
layout: case
title: "Critical F5 BIG-IP unauthenticated RCE Vulnerability"
excerpt: This vulnerability (CVE-2023-46747) may allow an unauthenticated adversary with network access to the BIG-IP system through the management port and/or self IP addresses to execute arbitrary system commands."
author: Boaz Braaksma
lead: Ralph Horn
status: Open
researchers:
- Axel Boesenach
- Ralph Horn
cves:
- CVE-2023-46747
products: BIG-IP (all modules)
versions:
- BIG-IP 17.1.0 - 17.1.1
- BIG-IP 16.1.0 - 16.1.4
- BIG-IP 15.1.0 - 15.1.10
- BIG-IP 14.1.0 - 14.1.5
- BIG-IP 13.1.0 - 13.1.5
recommendation: "Upgrade your affected versions to one of the hotfixes listed by F5 in their Security Advisory."

> patch_status: Full patched

Reviewer: ```suggestion
patch_status: Patch available
```

> -workaround:  "Restrict access to the Traffic Management User Interface (TMUI) from the internet."

Reviewer: ```suggestion
workaround:  "Restrict access to the Traffic Management User Interface (TMUI) from the internet."
```
Als je het minnetje niet weghaalt, dan wordt hij niet weergegeven.

status : Open
start: 2023-10-28
end:
timeline:
- start: 2023-10-26
  end:
  event: "F5 released hotfixes for BIG-IP versions 13.x through 17.x"
- start: 2023-10-28
  end:
  event: "DIVD started tracking this vulnerability"
- start: 2023-10-29
  end:
  event: "DIVD started researching fingerprint"
- start: 2023-10-29
  end:
  event: "DIVD identified vulnerable devices"
- start: 2023-10-31
  end:
  event: "DIVD started notifying stakeholders"
- start: 2023-11-02
  end:
  event: "First version of this casefile"
---

> ## Summary

Reviewer: ```suggestion
## Summary

```

> On October 26, 2023, F5 released security hotfixes for a critical unauthenticated RCE vulnerability in BIG-IP's Traffic Management User Interface (TMUI). This vulnerability is also tracked as CVE-2023-46747, with a CVSS v3.1 score of 9.8. This vulnerability is exploitable if the TMUI is exposed to the internet. A threat actor with network access to the vulnerable system could bypass the configuration utility authentication and execute arbitrary system command. On October 30, 2023, F5 updated the security advisory in order to warn about active exploitation in the wild.

Reviewer: ```suggestion
On October 26, 2023, F5 released security hotfixes for a critical unauthenticated RCE vulnerability in BIG-IP's Traffic Management User Interface (TMUI). This vulnerability is also tracked as {% cve CVE-2023-46747 %}, with a CVSS v3.1 score of 9.8. This vulnerability is exploitable if the TMUI (managmenet web interface) is exposed to the internet. A threat actor with network access to the vulnerable system could bypass the configuration utility authentication and execute arbitrary system commands. On October 30, 2023, F5 updated the security advisory in order to warn about active exploitation in the wild.
```
Ik mis 30-10-2023 in je tijdlijn.

> ## What you can do

Reviewer: ```suggestion

## What you can do

```

> For starters, it is recommended to restrict access to the Configuration Utility to only trusted networks or devices. F5 introduced Hotfixes in order to fix this issue. F5 provided as well a script that works as workaround to mitigate this vulnerability. This script should only be used if you are not able to apply the relevant security hotfix or you are not able to upgrade to a version that has a security hotfix. However, this script CANNOT be used on any BIG-IP versions prior to 14.1.0.

Reviewer: ```suggestion
For starters, it is recommended to restrict access to the Configuration Utility to only trusted networks or devices. This is in best practise that protects against vulnerabilities in this interface, which has historically proven to be a waeks spot.

F5 has released hotfixes that address this vulnerability. F5 also provides a script that works as workaround to mitigate this vulnerability. This script should only be used if you are not able to apply the relevant security hotfix or you are not able to upgrade to a version that has a security hotfix. However, this script CANNOT be used on any BIG-IP versions prior to 14.1.0.

```

> ## What we are doing

Reviewer: ```suggestion

## What we are doing

```
